src/dc_chat.rs

```diff
@@ -461,10 +463,13 @@ unsafe fn prepare_msg_common<'a>(
         );
         OK_TO_CONTINUE = false;
     }
+    dbg!(OK_TO_CONTINUE);
     if OK_TO_CONTINUE {
+        dbg!();
```

Comment:
Are these `dbg!`s going to be useful later?

Reply:
Oops. This is bug.